resources/qml/PrintMonitor.qml

```diff
@@ -44,7 +46,7 @@ Column
         delegate: Loader
         {
             sourceComponent: monitorItem
-            property string label: catalog.i18nc("@label", "Bed Temperature")
+            property string label: catalog.i18nc("@label", "Build plate")
```

Comment:
See UXD-80; "The text temperature (in Bed Temperature ) can be left out because the header is already called "Temperatures"."